util/evaluate_image_based.py

```diff
@@ -52,7 +52,7 @@
 
 from evaluate_plotting_helper import *
 
-torch.set_default_dtype(torch.float64)
+#torch.set_default_dtype(torch.float64)
```

Comment:
Is there a reason you've commented this line and then cast all the tensors individually later?

Reply:
great, thanks, good to have this documented

Reply:
Comment edited to provide this link